CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+### Added
+
+- Python 3.5, 3.6, 3.7 support
+
+### Changed
+
+- Memory files are always stored as binary streams and are encoded/decoded
+  based on the mode (r, rb, w, wb)
+- Switch to Python's standard zipfile libary
+- Python 2.5, 2.6 support
```

Comment:
@Pankrat did you mean to have a `### Removed` section or is there some word missing here?

Reply:
Indeed, I'll fix the changelog on master. Thanks for spotting this.